src/Note/Note.js

```diff
@@ -60,7 +60,7 @@ export default class Note extends React.Component {
             Modified
             {' '}
             <span className='Date'>
-              {format(modified, 'Do MMM YYYY')}
+            {format(new Date(modified), 'MM/dd/yyyy')}
```

Comment:
- Needed to be converted to date format
- Documentation to change display style: https://date-fns.org/v2.16.1/docs/format